packages/api-explorer/src/scenes/TypeTagScene/TypeTagScene.tsx

```diff
@@ -79,9 +93,20 @@ export const TypeTagScene: FC<TypeTagSceneProps> = ({ api }) => {
       </ButtonToggle>
       {Object.values(types).map(
         (type, index) =>
-          (value === 'ALL' ||
-            value === type.metaType.toString().toUpperCase()) && (
-            <Link key={index} to={buildTypePath(specKey, tag.name, type.name)}>
+          (selectedTagFilter === 'ALL' ||
+            selectedTagFilter === type.metaType.toString().toUpperCase()) && (
```

Comment:
I'm leaning towards not having this check. If user passes in an invalid value for a tag filter

Reply:
Did you mean to finish this comment? If a user passes in an invalid value for a tag filter, the expectation is that we handle this and default the user to have selected 'ALL', thus all of the options would show.